docs/deployment/landing_page.rst

```diff
@@ -3,6 +3,19 @@
 Landing Page
 ============
 
+SecureDrop itself is located on a Tor hidden service. Organizations also need to
+create a SecureDrop *Landing Page* that will explain how SecureDrop works, give
+sources instructions on how to access the Tor hidden service, and disclose the
+risks. We also recommend including a privacy policy (see our :ref:`Sample
+Privacy Policy`) describing what data is collected and how it will be used by
+your organization.
+
+.. important:: Implementing SecureDrop will bring more attention to your
+               organization from security researchers, hackers, and others. A
```

Comment:
do we want to include this scary reference to `hackers`. I don't want to scare monger potential news orgs. Additionally, the biggest risk is legal / national security isn't it?

Reply:
Maybe just "security researchers and others"?